db/migrate/20240102081043_create_contents.rb

```diff
@@ -4,7 +4,7 @@ def change
     create_table :contents do |t|
       t.string :title, null: false
       t.text :description, null: false
-      t.string :content_type, default: 'text'
+      t.bigint :content_type, default: 0
```

Comment:
do we need bigint here? or integer could be okay?

Reply:
I resolved it 
Thank you.